calculator.js

```diff
@@ -0,0 +1,24 @@
+var Calculator = function() {
+  this.display = 0
+  this.input
+}
+
+Calculator.prototype.getDisplay = function() {
+  return this.display
+}
+
+Calculator.prototype.keypress = function( value ) {
+  if ( value === 'AC' ) {
+    this.display = 0
+  }
```

Comment:
You may already be planning this, but I want to note it here.

This logic should eventually be moved out into its own function.  Keypress should determine what button or keyboard key was pressed, then call the appropriate function.